build_tools/github_actions/lint_buildifier.sh

```diff
@@ -20,16 +21,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Comment:
why this change?

Reply:
This was suggested by shellcheck: https://www.shellcheck.net/wiki/SC2155.

This is to avoid masking the return value of `cd`. Now that I think of it though, we have `set -o errexit`, so if `cd` fails the script will fail. I'll add a comment to suppress the lint.

Reply:
I probably prefer fixing things vs. suppression is my personal preference since code tends to be copied.

Reply:
Thanks for commenting, I actually misunderstood the lint, and when I tried to test the behavior of `readonly` with `set -o errexit` I didn't write my test correctly. The lint is correct: `readonly` will cause errors to be swallowed and the script will continue execution, even if `errexit` is set. I removed the suppressions and fixed the code.